examples/minimal/packages/contracts/script/PostDeploy.s.sol

```diff
@@ -12,6 +12,9 @@ import { ChatNamespacedSystem } from "../src/systems/ChatNamespacedSystem.sol";
 
 contract PostDeploy is Script {
   function run(address worldAddress) external {
+    // Specify a store so that you can use tables directly in PostDeploy
+    StoreSwitch.setStoreAddress(worldAddress);
```

Comment:
looks like this is missing an import of `StoreSwitch`

Reply:
Sorry, fixed it. 🙏

Reply:
Sorry, fixed it. 🙏